aea/crypto/ethereum.py

```diff
@@ -192,7 +192,7 @@ def send_transaction(
         destination_address: AddressLike,
         amount: int,
         tx_fee: int,
-        chain_id: int = 1,
+        chain_id: int = 3,
```

Comment:
I think we should add tx_nonce as a keyword argument here

Reply:
The thing here is if I pass it as reference we will have to change a big part of the code to support the keyword. Since we will have to pass it from the handlers to decision maker and then ethereum crypto. The same way we are passing the info.

Reply:
We should evaluate this together. For our usecases we will always want the tx_nonce to be set. Otherwise we cannot disambiguate. So having it in kwargs can cause issues with it being missing.

Reply:
In TransactionMessage I'd add
`
assert isinstance(self.tx_nonce, str), "Tx_nonce must be of type str."
assert len(self.body) == 12
`
then the skill can send the tx_nonce to the DM this way. Then in all the methods which require the tx_digest, we add it to the list of arguments.